declarative_clusterconfig/customizing-permissions-by-creating-user-defined-cluster-roles-for-cluster-scoped-instances.adoc

```diff
@@ -0,0 +1,72 @@
+= Customizing permissions by creating user-defined cluster roles for cluster-scoped instances
+:context: customizing-permissions-by-creating-user-defined-cluster-roles-for-cluster-scoped-instances
+
+toc::[]
+
+For the default cluster-scoped instance, the {gitops-title} Operator grants additional permissions for managing certain cluster-scoped resources. Consequently, as a cluster administrator, when you deploy an Argo CD as a cluster-scoped instance, the Operator creates additional cluster roles and cluster role bindings for the {gitops-shortname} control plane components. These cluster roles and cluster role bindings provide the additional permissions that Argo CD requires to operate at the cluster level. 
+
+If you do not want the cluster-scoped instance to have all the Operator-given permissions and choose to add or remove permissions to cluster-wide resources, you must first disable the creation of the default cluster roles for the cluster-scoped instance. Then, you can customize permissions for the following cluster-scoped instances:
+
+* Default ArgoCD instance (default cluster-scoped instance)
+* User-defined cluster-scoped Argo CD instance
+
+This guide provides instructions with examples to help you create a user-defined cluster-scoped Argo CD instance, deploy an Argo CD application in your defined namespace that contains custom configurations for your cluster, disable the creation of the default cluster roles for the cluster-scoped instance, and customize permissions for user-defined cluster-scoped instances by creating new cluster roles and cluster role bindings for the {gitops-shortname} control plane components. 
+
+[NOTE]
+====
+As a developer, if you are creating an Argo CD application and deploying cluster-wide resources, ensure that your cluster administrator grants the necessary permissions to them.
+
+Otherwise, after the Argo CD reconciliation, you will see an authentication error message in the application's `Status` field similar to the following example:
+
+.Example authentication error message
+
+[source,terminal]
+----
+persistentvolumes is forbidden: User "system:serviceaccount:gitops-demo:argocd-argocd-application-controller" cannot create resource "persistentvolumes" in API group "" at the cluster scope.
+----
+====
+
+[id="prerequisites_customizing-permissions-by-creating-user-defined-cluster-roles-for-cluster-scoped-instances"]
+== Prerequisites
+
+* You have installed {gitops-title} 1.13.0  or a later version on your {OCP} cluster.
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the {gitops-title} `argocd` CLI.
+* You have installed a xref:../declarative_clusterconfig/configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations.adoc#using-argo-cd-instance-to-manage-cluster-scoped-resources_configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations[cluster-scoped Argo CD instance] in your defined namespace `spring-petclinic`. 
```

Comment:
Does the user have to use this particular namespace, or are you providing it as an example? If the latter, perhaps revise this as follows:
```suggestion
* You have installed a xref:../declarative_clusterconfig/configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations.adoc#using-argo-cd-instance-to-manage-cluster-scoped-resources_configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations[cluster-scoped Argo CD instance] in your defined namespace. For example, `spring-petclinic`.
```